released/discovery_center/mission_3501/step1/main.tf

```diff
@@ -345,64 +347,95 @@ resource "btp_subaccount_role_collection" "alm_ts_import_operator" {
 # Prepare and setup app: SAP Build Workzone, standard edition
-###############################################################################################
+# ------------------------------------------------------------------------------------------------------
 # Entitle subaccount for usage of app  destination SAP Build Workzone, standard edition
 resource "btp_subaccount_entitlement" "build_workzone" {
-  subaccount_id = data.btp_subaccount.project.id
+  subaccount_id = data.btp_subaccount.dc_mission.id
   service_name  = local.service_name__build_workzone
   plan_name     = var.service_plan__build_workzone
   amount        = var.service_plan__build_workzone == "free" ? 1 : null
 }
 
 # Create app subscription to SAP Build Workzone, standard edition (depends on entitlement)
 resource "btp_subaccount_subscription" "build_workzone" {
-  subaccount_id = data.btp_subaccount.project.id
+  subaccount_id = data.btp_subaccount.dc_mission.id
   app_name      = local.service_name__build_workzone
   plan_name     = var.service_plan__build_workzone
   depends_on    = [btp_subaccount_entitlement.build_workzone]
 }
 
 # Assign users to Role Collection: Launchpad_Admin (SAP Build Workzone, standard edition)
 resource "btp_subaccount_role_collection_assignment" "launchpad_admin" {
-  for_each             = toset("${var.workzone_se_administrators}")
-  subaccount_id        = data.btp_subaccount.project.id
+  for_each             = toset(var.workzone_se_administrators)
+  subaccount_id        = data.btp_subaccount.dc_mission.id
   role_collection_name = "Launchpad_Admin"
+  origin               = local.origin_key_app_users
   user_name            = each.value
   depends_on           = [btp_subaccount_subscription.build_workzone]
 }
 
+# ------------------------------------------------------------------------------------------------------
+# Create tfvars file for step 2 (if variable `create_tfvars_file_for_step2` is set to true)
+# ------------------------------------------------------------------------------------------------------
+resource "local_file" "output_vars_step1" {
+  count    = var.create_tfvars_file_for_step2 ? 1 : 0
+  content  = <<-EOT
+      globalaccount        = "${var.globalaccount}"
+      cli_server_url       = ${jsonencode(var.cli_server_url)}
+      custom_idp           = ${jsonencode(var.custom_idp)}
+
+      subaccount_id        = "${data.btp_subaccount.dc_mission.id}"
+
+      cf_api_url           = "${jsondecode(btp_subaccount_environment_instance.cloudfoundry.labels)["API Endpoint"]}"
+
+      cf_org_id            = "${jsondecode(btp_subaccount_environment_instance.cloudfoundry.labels)["Org ID"]}"
+      cf_org_name          = "${jsondecode(btp_subaccount_environment_instance.cloudfoundry.labels)["Org Name"]}"
+
+      cf_space_name        = "${var.cf_space_name}"
+
+      cf_org_admins        = ${jsonencode(var.cf_org_admins)}
+      cf_space_developers  = ${jsonencode(var.cf_space_developers)}
```

Comment:
missing cf_org_users, pls add this line here:
cf_org_users        = ${jsonencode(var.cf_org_users)}